mindmeld/components/classifier.py

```diff
@@ -216,15 +218,14 @@ def fit(self, queries=None, label_set=None, incremental_timestamp=None, **kwargs
             label_set = model_config.train_label_set
             label_set = label_set if label_set else DEFAULT_TRAIN_SET_REGEX
 
-        new_hash = self._get_model_hash(model_config, queries, label_set)
+        new_hash = self._get_model_hash(model_config, label_set)
         cached_model = self._resource_loader.hash_to_model_path.get(new_hash)
 
         if incremental_timestamp and cached_model:
-            logger.info("No need to fit. Loading previous model.")
-            self.load(cached_model)
-            return
+            logger.info("No need to fit.  Previous model is cached.")
```

Comment:
Does this mean in order to parse queries, one has to do this:
```
nlp.fit()
nlp.load()
nlp.process("xyz")
```

instead of:
```
nlp.fit()
nlp.process("xyz")
```
If this is true, just wondering if the gains we see here can be done without a change in the API behavior since this would be a backwards incompatible change.

Reply:
I guess I'm not sure what interfaces are external.  The nlp.build function loads all of the models into memory once it completes.   If people use fit() on individual classifiers independently of the nlp.build() method, we could have an option to load it regardless to make it backwards compatible.

Reply:
I committed a change to handle this.